client/src/use/useImageEnhancements.ts

```diff
@@ -0,0 +1,54 @@
+
+import {
+  computed,
+  ref, Ref, set as VueSet,
+} from '@vue/composition-api';
+
+// Expecting this may be a placeholder for more complicated client side enhancements
+// or more image filters
+export interface ImageEnhancements {
+    blackPoint?: number;
+    whitePoint?: number;
+  }
+
+  /**
+   * Modified markChangesPending for image Enhancments specifically when it is saved per dataset
+   */
+  interface ImageEnhancementParams {
+    markChangesPending: () => void;
+  }
+
+
+export default function useImageEnhancements({ markChangesPending }: ImageEnhancementParams) {
+  const imageEnhancements: Ref<ImageEnhancements> = ref({});
+
+  const setSVGFilters = ({ blackPoint, whitePoint }:
+    {blackPoint?: number; whitePoint?: number }) => {
+    VueSet(imageEnhancements.value, 'blackPoint', blackPoint);
+    VueSet(imageEnhancements.value, 'whitePoint', whitePoint);
+    markChangesPending();
```

Comment:
This composition function should not call markChangesPending.  There's nothing to save.  It should probably not take it as a param.

Reply:
:facepalm: -  I was in the middle of adding saving of the metadata, and I must have forgot to remove it.